### ISSUE TITLE
Shortcuts for Move up (T) / Move down (Y) are bad as T and Y are neighboring keys only on QWERTY keyboards - Better use arrow keys plus modifiers

### ISSUE THREAD

**Issue by porg** [WordPress/gutenberg#51645]
- #10455 seems to have been done through an Anglosaxon perspective without i18n in mind
- It assigned "Move up" to `⇧-⌥-⌘-T` and "Move down" to `⇧-⌥-⌘-Y`
  - Putting keyboard shortcut counterparts/antonyms as keyboard neighbors can make sense
  - But the T and Y key are neighbors only on a QWERTZ keyboard layout
- Also the shortcuts for "Move up/down/left/right" should rather be combined with the arrow keys
  - Because "move cursor" / "move selection" / "extend selection" are conceptually related tasks, likely conducted sequentially, hence using the arrow keys and just varying the modifier keys is established best practise.
  - No need to re-invent the wheel, as best practise take macOS Cocoa native text selection commands, outliner/mindmap software for indenting/outdenting/grouping and also graphics/layout software.

**Comment by porg** [comment]
Hi,
I ping you, as you were originally involved:
@ZebulanStanphill as you opened #10455
@ntsekouras as you coded #23276

**Comment by hanneslsm** [comment]
Oh, wow. It never came to my mind that T and Y could be neighbouring keys. I always struggled with remembering the shortcuts lol

When tackling this issue, we should not forget that `Add before` & `Add after` have similar shortcuts.

**Comment by porg** [comment]
- As an experienced UX designer I have seen bad i18n far too often, hence easily spotted the root cause for this.
- Yes ofc `Add before` & `Add after` must holistically be integrated into this whole shortcut set.

**Comment by talldan** [comment]
I don't think it was possible to use arrow keys, otherwise arrow keys would definitely have been used.

WordPress has generally taken the approach of not overriding existing text manipulation, browser and OS shortcuts, and unfortunately that leaves very few options. I think arrow keys are almost entirely reserved for text manipulation or OS level shortcuts.

**Comment by porg** [comment]
1. Not overriding keyboard shortcuts and not overriding the right click context-menu leaves only the possibilities for a rather mediocre UX on Desktop devices, given that the Block Editor already has and foresee-able will get quite many shortcuts. Many web apps dare to do this and do this really well. And feel very native and efficient that way.

2. Also to strive for cross platform consistency regarding shortcuts is ok. But not at all cost. In some regards the differences should be respected.
    - For an individual user it is more beneficial if shortcuts are consistent with the platform and other apps they use on a daily basis (high probability)
    - …rather than being able to operate WordPress when cycling between different Desktop OS (far less likely).

**Comment by Mamaduka** [comment]
I think the current shortcuts were chosen because of the existing 'Add before/after' commands. It might be a good time to revisit these and make them more intuitive.

Using the existing key combo plus arrow keys seems like a good option, though it might make shortcut logic more complex if we account for the `orientation` setting (vertical|horizontal).

cc @annezazu, @mtias

**Comment by annezazu** [comment]
This touches on the following as well which might be a better and more comprehensive approach to take: https://github.com/WordPress/gutenberg/issues/3218
